### DISCUSSION TITLE
Daily 20/09

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#72] [Daily]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué hice o haré hoy?
- Impedimentos o dificultades

**Comment by NataliaAlvarezIspc** [comment]
Ayer participé en la clase de la tarde de app móviles. Por la noche nos reunimos para acordar como seguimos, asignarnos tareas y ponernos de acuerdo en el esqueleto en android studio para subir. Ya me bajé el repositorio y pude abrir el proyecto en AS.
Hoy voy a revisar la documentación, subir el diagrama de clases al repo. Por la noche tengo clases de inglés, así que tengo que dedicarle tiempo a estudiar antes de la clase.

**Comment by BraianTroncoso** [comment]
Anoche participé en la reunion para saber como seguir con el proyecto y participé de la clase del ISPC. Subimos tambien el esqueleto de la app y configuramos algunas cosas del proyecto para que esté por default. Hoy voy a hacer unos cuestionarios de ciberseguridad, y tengo que seguir estudiando Java.

**Comment by MarielaGomez** [comment]
Ayer completé los cuestionarios de ciberseguridad y adelanté lectura, continúo practicando para android.
